sites/feedback/reports/acr-2025-06.json

```diff
@@ -124,28 +267,60 @@
       "earl:test": [ "WCAG21:status-messages" ],
       "acr:severity": "acr:notRelevant",
       "acr:relevancy": "wbv:acr#comments"
+    },
+    {
+      "@type": "wf:Task",
+      "dct:title": "Improve WET-BOEW Postback plugin",
+      "wf:goalDescription": "The WET-BOEW Postback plugin should be improved so that the confirmation and error messages are announced by all popular screen-readers. The current solution only displays the message which is not announced by JAWS or NVDA because no text change happened.",
+      "dct:created": "2025-06-11",
+      "acr:requirement": "WCAG21:status-messages",
+      "earl:test": [ "WCAG21:status-messages" ],
+      "acr:severity": "acr:moderate",
+      "acr:relevancy": "wbv:acr#usability"
+    },
+    {
+      "@type": "wf:Task",
+      "dct:title": "Add focus to text box on click of No button",
+      "wf:goalDescription": "When users click the No button, give the focus to the newly displayed text box so that users don't have to manually move to it.",
```

Comment:
I am not sure about that behaviour. Do we have some reference? Alternatively, you can mention that this need to be researched first and evaluated if a fix that move the focus are required.
Note: I didn't find any WCAG technique mentioning such behaviour via the SC this concern was reported in the assessment.
You can also adjust the severity, I don't think "moderate" is right unless there is evidence. Idem for the relevancy, that look like to be in between a comment to a opinionated action. see: https://wet-boew.github.io/vocab/acr#relevancy